Mf2/Parser.php

```diff
@@ -649,7 +667,8 @@ public function parseU(\DOMElement $u) {
 		} else {
 			$uValue = $this->textContent($u);
 		}
-				return $this->resolveUrl($uValue);
+
+		return $this->resolveUrl($uValue);
```

Comment:
I think calling `resolveUrl` can be wrapped in a condition checking if `$uValue` is a string, since `resolveUrl` is already called in the new img+alt parsing method.